griptape/drivers/prompt/amazon_sagemaker_prompt_driver.py

```diff
@@ -43,3 +43,6 @@ def try_run(self, prompt_stack: PromptStack) -> TextArtifact:
             return self.prompt_model_driver.process_output(decoded_body)
         else:
             raise Exception("model response is empty")
+
+    def try_stream(self, _: PromptStack) -> Iterator[TextArtifact]:
```

Comment:
Should we add a validation for the `stream` property instead, so it doesn't fail at runtime?